tst/working/veryslow/MatFDPM.tst

```diff
@@ -1,33 +1,42 @@
 # FDPM = fully deleted permutation module
-
-# Alternating:
-gap> deg := Random(10,88);;
-gap> g := AlternatingGroup(deg);;
-gap> f := GF(Random([2,3,4,5,7,9,11]));;
-gap> gens := List([1..5],x->PseudoRandom(g));;
-gap> mgens := List(gens,x->PermutationMat(x,deg,f));;
-gap> m := GModuleByMats(mgens,f);;
-gap> cf := MTX.CompositionFactors(m);;
-gap> dims := List(cf,x->x.dimension);;
-gap> max := Maximum(dims);;
-gap> pos := Position(dims,max);;
-gap> x := PseudoRandom(GL(max,f));;
-gap> mgens := List(cf[pos].generators,y->y^x);;
-gap> h := Group(mgens);;
-gap> ri := RECOG.TestGroup(h,false,Size(g));;
-
 # Symmetric:
-gap> deg := Random(10,88);;
-gap> g := SymmetricGroup(deg);;
-gap> f := GF(Random([2,3,4,5,7,9,11]));;
-gap> gens := List([1..5],x->PseudoRandom(g));;
-gap> mgens := List(gens,x->PermutationMat(x,deg,f));;
-gap> m := GModuleByMats(mgens,f);;
-gap> cf := MTX.CompositionFactors(m);;
-gap> dims := List(cf,x->x.dimension);;
-gap> max := Maximum(dims);;
-gap> pos := Position(dims,max);;
-gap> x := PseudoRandom(GL(max,f));;
-gap> mgens := List(cf[pos].generators,y->y^x);;
-gap> h := Group(mgens);;
-gap> ri := RECOG.TestGroup(h,false,Size(g));;
+gap> testFDPM := function(deg, field, alternating)
+> local g, gens, mgens, m, cf, dims, max, pos, x, h, result;
+> if alternating then
+>   g := AlternatingGroup(deg);
+> else
+>   g := SymmetricGroup(deg);
+> fi;
+> gens := List([1..5],x->PseudoRandom(g));
+> mgens := List(gens,x->PermutationMat(x,deg,field));
+> m := GModuleByMats(mgens,field);
+> cf := MTX.CompositionFactors(m);
+> dims := List(cf,x->x.dimension);
+> max := Maximum(dims);
+> pos := Position(dims,max);
+> x := PseudoRandom(GL(max,field));
+> mgens := List(cf[pos].generators,y->y^x);
+> h := Group(mgens);
+> return Size(RecogniseGroup(h)) = Size(g);
+> #return RECOG.TestGroup(h,false,Size(g));
+> end;;
+gap> l := [];
+gap> for deg in [10, 13, 17, 32] do
+> for field in List([2,3,4,7,9], GF) do
+> Print(deg, field);
+> Add(l, testFDPM(deg, field, false));
+> Add(l, testFDPM(deg, field, true));
+> od;
+> od;
+gap> func := function(l)
+> for deg in [10, 13, 17] do
+> for field in List([2,3,4,7,9], GF) do
+> Add(l, testFDPM(deg, field, false));
+> Add(l, testFDPM(deg, field, true));
+> od;
+> od;
```

Comment:
The loops throw an error on my local machine, since the expected output are lines of the format:
```
 F dim=   8 field=2  0      0 
```